$NetBSD: patch-ad,v 1.2 2008/11/23 23:40:12 heinz Exp $

  - Honor configure option --with-embedded-src-path
    (already committed in unstable version upstream)

--- closure.c.orig	2008-03-04 10:01:15.000000000 +0000
+++ closure.c
@@ -174,10 +174,13 @@ static void get_base_dirs()
    char *appdata;
 #endif
 
-   /*** The source directory is supposed to hold the most recent files,
+   /*** Unless completely disabled through a configure option, the
+	source directory is supposed to hold the most recent files,
 	so try this first. Not necessary under Windows as it will always
 	use the directory the binary has been called from. */
 
+#ifdef WITH_EMBEDDED_SRC_PATH_YES
+
 #ifndef SYS_MINGW
    if(!stat(SRCDIR, &mystat))
    {  Closure->binDir = g_strdup(SRCDIR);
@@ -187,6 +190,8 @@ static void get_base_dirs()
    } 
 #endif
 
+#endif /* WITH_EMBEDDED_SRC_PATH_YES */
+
    /*** Otherwise try the installation directory. 
 	On Unices this is a hardcoded directory.
 	Windows has binary distributions with no prior known installation place,
